### PR TITLE
Use libgl-devel and libegl-devel

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
I've been struggling on the build here https://github.com/conda-forge/qt-main-feedstock/pull/282 ... just fyi.

**Comment by ehfd** [comment]
The Linux build is successful here and there's an issue with Travis, while https://github.com/conda-forge/qt-main-feedstock/pull/282 has a link error, so I don't think both are related.

File: recipe/yum_requirements.txt [hunk 1]
```diff
@@ -1,6 +1,3 @@
 mesa-libGL
-mesa-libGL-devel
-mesa-libEGL-devel
 mesa-dri-drivers
```
Comment (hmaarrfk) [inline recipe/yum_requirements.txt:2]: Why can’t all of these be removed too?

Comment (ehfd) [reply recipe/yum_requirements.txt:2]: They can and should.

Comment (ehfd) [reply recipe/yum_requirements.txt:2]: All of them should be removed.

Comment (Tobias-Fischer) [reply recipe/yum_requirements.txt:2]: I also believe that we need a rerender for these changes to take effect, don't we?

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: recipe/yum_requirements.txt (hunk 1 above)
Comment (isuruf) [reply recipe/yum_requirements.txt:2]: I guess the tests don't need any EGL implementation. If they did, we would need them

**Comment by hmaarrfk** [comment]
thanks isuru. we got this into traversaro's PR for the 5.15 branch

**Pull request closed without merging**